tox.ini

```diff
@@ -1,35 +1,69 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
+minversion = 3.18
 envlist =
-    py{27,py,35,36,37,38}{,-subunit},coverage,docs
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
+commands =
+    zope-testrunner --test-path=src {posargs:-vc}
 extras =
     test
-    subunit: subunit
+
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
 commands =
-    python setup.py -q test -q
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
 
 [testenv:coverage]
-usedevelop = true
-basepython =
-    python3
-commands =
-    coverage run setup.py -q test -q
-    coverage combine
-    coverage report --fail-under=85
-setenv =
-    COVERAGE_PROCESS_START = {toxinidir}/.coveragerc
-extras =
-    test
-    subunit
```

Comment:
and this

Reply:
this could also explain the coverage percentage drop!

Reply:
huh, re-enabling the `subunit` extra for tox -e coverage brought our percentage back to 85%!